examples/advanced/experiment-tracking/README.md

```diff
@@ -32,35 +95,134 @@ FLARE allows seamless switching between centralized and decentralized experiment
 ### MLflow
 
-The `mlflow` folder contains [examples](./mlflow/README.md) showcasing the usage of the capability to stream from
-the clients to the server to write to an MLflow tracking server, as well as concurrent streaming to Tensorboard.
-It also illustrate how to stream the metrics to site-specific mlflow server.
+The `mlflow` folder contains [three examples](./mlflow/README.md) demonstrating different MLflow configurations:
+
+1. **`hello-pt-mlflow`**: Server-side centralized tracking
+2. **`hello-pt-mlflow-client`**: Site-specific decentralized tracking
+3. **`hello-lightning-mlflow`**: PyTorch Lightning integration with MLflow
+
+**Key Features**:
+- Flexible server-side or client-side tracking
+- Experiment and run management
+- Automatic metric logging and artifact storage
 
 ### Weights and Biases
 
 The `wandb` folder contains the [Hello PyTorch with Weights and Biases](./wandb/README.md) example
-showing how Weights and Biases can be used as a back end for the `WandBReceiver` to write to.  
+showing how to use W&B for experiment tracking with both server-side and client-side options.
+
+**Key Features**:
+- Online/offline mode support
+- Rich experiment dashboards
+- Configurable server or client-side tracking
+
+---
+
+## Quick Start Guide
+
+### 1. Install Requirements
+
+```bash
+cd examples/advanced/experiment-tracking/<framework>
+pip install -r requirements.txt
+```
+
+### 2. Download Data (if needed)
+
+```bash
+./prepare_data.sh
+```
+
+### 3. Run Example
+
+```bash
+cd <framework>/jobs/<job_name>/code
+python job.py
+```
+
+### 4. View Results
+
+**TensorBoard**:
+```bash
+tensorboard --logdir=/tmp/nvflare/jobs/workdir/server/simulate_job/tb_events
+```
+
+**MLflow**:
+```bash
+mlflow ui --backend-store-uri /tmp/nvflare/jobs/workdir/server/simulate_job/mlruns
+```
+
+**Weights & Biases**:
+Visit your W&B dashboard at https://wandb.ai
+
+---
+
+## Adding Tracking to Your Own Recipe
+
+### Server-Side Tracking (Centralized)
+
+```python
+from nvflare.recipe.utils import add_experiment_tracking
+
+# After creating your recipe
+add_experiment_tracking(recipe, "mlflow", tracking_config={
+    "tracking_uri": "file:///tmp/mlruns",
+    "kw_args": {
+        "experiment_name": "my-experiment",
+        "run_name": "my-run",
+    }
+})
+```
+
+### Client-Side Tracking (Decentralized)
+
+```python
+from nvflare.app_opt.tracking.mlflow.mlflow_receiver import MLflowReceiver
+
+# Add tracking to specific clients
+for site_name in ["site-1", "site-2"]:
+    receiver = MLflowReceiver(
+        tracking_uri=f"file:///tmp/{site_name}/mlruns",
+        kw_args={"experiment_name": f"{site_name}-experiment"}
+    )
+    recipe.job.to(receiver, site_name, id="mlflow_receiver")
+```
+
+---
+
+## Additional Resources
 
+- [Experiment Tracking Programming Guide](https://nvflare.readthedocs.io/en/main/programming_guide/experiment_tracking.html)
+- [Recipe API Documentation](https://nvflare.readthedocs.io/en/main/programming_guide/job_recipes.html)
```

Comment:
we dont have this link,
need to change to https://nvflare.readthedocs.io/en/main/user_guide/data_scientist_guide/job_recipe.html